schemas/accelerator.sql

```diff
@@ -0,0 +1,218 @@
+-- This schema adds the tables necessary for Accelerator accounting as a
+-- separate record as part of the wider Cloud Accounting system.
+
+-- ------------------------------------------------------------------------------
+DROP FUNCTION IF EXISTS SiteNameLookup;
+DELIMITER //
+CREATE FUNCTION SiteNameLookup(lookup INTEGER) RETURNS VARCHAR(255) DETERMINISTIC
+BEGIN 
+    DECLARE result VARCHAR(255);
+    SELECT name FROM Sites WHERE id = lookup INTO result;
+    RETURN result;
+END //
+DELIMITER ;
+
+DROP TABLE IF EXISTS AcceleratorRecords;
+CREATE TABLE AcceleratorRecords (
+  UpdateTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  MeasurementMonth INT NOT NULL,
+  MeasurementYear INT NOT NULL,
+
+  AssociatedRecordType VARCHAR(255) NOT NULL,
+  AssociatedRecord VARCHAR(255) NOT NULL,
+
+  GlobalUserName VARCHAR(255),      
+  FQAN VARCHAR(255) NOT NULL,       
+  SiteID INT NOT NULL,
+  Count DECIMAL(10,3) NOT NULL,
+  Cores INT,
+  ActiveDuration INT,
+  AvailableDuration INT,
```

Comment:
This should not be nullable if this is a mandatory field - also, within the acceleratorrecords schema I see inconsistencies to the mandatory fields in the acceleratorrecord messaging, could we discuss?

Reply:
I have raised this PR to address a few issues with the code logic, etc. I have not worked with the team to determine which fields are mandatory or not. Sure, we can have a discussion.